drivers/modem/modem_cellular.c

```diff
@@ -1888,6 +1910,10 @@ MODEM_CHAT_SCRIPT_CMDS_DEFINE(telit_me910g1_init_chat_script_cmds,
 				  MODEM_CHAT_SCRIPT_CMD_RESP_NONE("AT", 100),
 				  MODEM_CHAT_SCRIPT_CMD_RESP_NONE("AT", 100),
 				  MODEM_CHAT_SCRIPT_CMD_RESP("ATE0", ok_match),
+				  MODEM_CHAT_SCRIPT_CMD_RESP("AT+ICCID", iccid_match),
```

Comment:
Funny, the guide I found lists this command as `AT+QCCID`
https://www.quectel.com/wp-content/uploads/2021/03/Quectel_BG95BG77BG600L_Series_AT_Commands_Manual_V2.0-3.pdf

For the HL7800 it's `AT+CCID`. I love how we can't use the same command across modems for this one...
Also for the HL7800, if the SIM is an ESIM, it can also return and EID. Can the BG95 do that too?
https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/modem/hl7800.c#L1809

Reply:
Gotta love it. /s

I only have the Telit ME910 to test with here, I only updated the BG95 because it's used for the testsuite, but it looks like that might be wrong as you pointed out!